src/geneweaver/api/services/genes.py

```diff
@@ -56,7 +56,7 @@ def get_gene_preferred(cursor: Cursor, gene_id: int) -> dict:
         logger.error(err)
         raise err
 
-    return {"gene": gene}
+    return {"data": gene}
```

Comment:
I think this endpoints only ever returns a single Gene, so in that case we want to directly return the Gene object.